README.md

```diff
@@ -39,10 +39,10 @@ RSocket interface contains 5 methods:
   `suspend fun metadataPush(metadata: ByteReadPacket)`
 
 ## Using in your projects
-The `master` branch is now dedicated to development of multiplatform rsocket-kotlin.
-For now only snapshots are available via [oss.jfrog.org](oss.jfrog.org) (OJO).
+The `master` branch is now dedicated to development of multiplatform rsocket-kotlin. For now only snapshots are available
+via [oss.jfrog.org](oss.jfrog.org) (OJO).
```

Comment:
This link is a dead link

Reply:
Maybe link to https://oss.jfrog.org/artifactory/oss-snapshot-local/io/rsocket/kotlin/ with an example for using a snapshot.

Reply:
will do